code/game/objects/items/weapons/clown_items.dm

```diff
@@ -100,3 +100,12 @@
 		spawn(20)
 			spam_flag = 0
 	return
+
+/obj/item/weapon/bikehorn/afterattack(atom/target, mob/user as mob)
+	if (spam_flag == 0)
+		spam_flag = 1
+		playsound(get_turf(src), 'sound/items/bikehorn.ogg', 50, 1)

```

Comment:
Okay so this is fairly silly, attacks should already be delayed

Do something like
>/obj/item/weapon/bikehorn/afterattack(atom/target, mob/user as mob, proximity_flag)
if(proximity_flag && istype(target,/mob)
playsound(get_turf(src), 'sound/items/bikehorn.ogg', 50, 1)